x-pack/filebeat/module/cisco/asa/test/additional_messages.log-expected.json

```diff
@@ -141,7 +143,8 @@
         "log.level": "informational",
         "log.offset": 324,
         "network.direction": "inbound",
-        "network.protocol": "icmp",
+        "network.iana_number": 1,
+        "network.transport": "icmp",
```

Comment:
Moving this field from `network.protocol` to `network.transport` can be a breaking change. Is this expected?

@P1llus wdyt?

Reply:
Not sure if it is a breaking change. I was parsing according to ECS field reference, ICMP is better mapped to network.transport since it has it's own IANA number.  Also, it's not technically an L7 protocol like HTTP or FTP, which is how network.protocol is defined in ECS.

Reply:
Yeah you are correct on that @hitchatwork , it's not that your mapping was wrong, it's just that if the current mapping was indeed incorrect, fixing it would mean that other users that might already be using the field, it would break for them.
Might need @andrewkroh or @adriansr to pitch in, in terms of how we want to apply this.

Reply:
@P1llus - I could change it so that it matches what is currently set, so this doesn't sit in review purgatory.  After this is reviewed and if commited, I can open a separate PR to "fix" the classification of ICMP from protocol to transport.  LMK.  Thanks!

Reply:
I think taking it in two separate PRs would make it easier to push the rest of the changes, though just wanted to see if anyone else would pitch in as well

Reply:
@P1llus - I just created a 'set' clause if the network.tranport was set to 'icmp', then set the network.protocol to be 'icmp'.  That should prevent any breaking changes and help improve the consistency of nomenclature with ECS.

Reply:
I reset the ICMP fields to what they were originally.  Will create a separate PR for fixing ICMP classification as transport instead of protocol.